### DISCUSSION TITLE
How to move firmware target form Embargo to the Test ?

### DISCUSSION THREAD

**Discussion by jyu99** [fwupd/fwupd#4192] [Q&A] [answered]
What is the condition about the firmware from embargo to the test ?

I have done for Embargo test and get the success report in my account, so what is my next step ?

**Comment by hughsie** [comment, marked as answer]
Three things have to be true:

* You have to have the "Trusted" ACL on your user account (e.g. "can move to testing and stable")
* The vendor the firmware is owned by has to have provided the legal documentation to "unlock" the account
* The firmware must have no "problems" (e.g. missing update description) that prevent it being moved to testing or stable

If you can send me an email with the link to the firmware then I can take a look and give you specific advice.
